ci-operator/step-registry/optional-operators/ci/run-bundle/optional-operators-ci-run-bundle-ref.yaml

```diff
@@ -0,0 +1,36 @@
+ref:
+  as: optional-operators-ci-run-bundle
+  from_image:
+    namespace: ci
+    name: cvp-operator-scorecard
+    tag: v1
+  commands: optional-operators-ci-run-bundle-commands.sh
+  resources:
+    requests:
+      cpu: 300m
+      memory: 300Mi
+  dependencies:
+  - name: bundle-image
+    env: OO_BUNDLE_IMG
+  env:
+  - name: OO_INSTALL_NAMESPACE
+    default: "!create"
+    documentation: |
+      This namespace must already exist in the cluster.
+      This is an optional field which will default to the kubeconfig context if not provided.
+  - name: OO_TARGET_NAMESPACES
+    default: "!all"
+    documentation: |
+      The default value `!all` means the Operator will watch all namespaces (cluster-scoped Operators).
+      If the value is `!own`, then the Operator will watch its own namespace (from namespace or the kubeconfig default).
+      If the value is `!single`, the the Operator will watch a namespace, not necessarily its own.
+      This is an optional parameter, but if the CSV does not support AllNamespaces,
+      then this parameter becomes required to instruct run bundle with the appropriate target namespace.
+  - name: OO_INDEX_IMG
+    default: "quay.io/operator-framework/upstream-opm-builder:latest"
+    documentation: |
+      This specifies an index image in which to inject the given bundle.
+      This is an optional field which will default to quay.io/operator-framework/upstream-opm-builder:latest
```

Comment:
No need to specify the parameter is optional (thats implied by it having a default value) nor to duplicate it default value (there's a structured field for that).